pkg/eraser/eraser.go

```diff
@@ -198,6 +248,24 @@ func main() {
 		os.Exit(1)
 	}
 
+	// read excluded values from excluded configmap
```

Comment:
I would put this in an `init()` function, since its purpose is to initialize a `var`.